dpxdt/client/capture.js

```diff
@@ -77,6 +76,12 @@ if (config.cookies) {
     });
 }
 
+// Add username and password as a parameter for HTTP basic auth
+if (config.httpUserName && config.httpPassword) {
```

Comment:
Oh cool I meant for you to add it here to the documentation of `config.json` <https://github.com/bslatkin/dpxdt#format-of-config>